### ISSUE TITLE
Allow multiuser admin to set a new Default Quota

### ISSUE THREAD

**Issue by andrewxhill** [CartoDB/cartodb#4466]
They admin of a 5k org should be able to set a Default Quota that is custom to their own needs. Including various flags on/off etc.

**Comment by juanignaciosl** [comment]
You can already set the default quota for new users. I guess you mean defaults for:

- Extra geocoding.
- Extra tweets.

Am I missing something?

**Comment by andrewxhill** [comment]
match the superadmin probably. 

* soft quota (on each geocoding/twitter)?
* private data/tables?

**Comment by juanignaciosl** [comment]
Uhm. Private datatables is not something an org owner can set right now. We should add it to user edition level (that toggle is copied from owner's at user creation time).

**Comment by juanignaciosl** [comment]
@andrewxhill are you ok with this set of defaults?


![screen shot 2015-07-28 at 13 09 16](https://cloud.githubusercontent.com/assets/932968/8929978/ee0161dc-3529-11e5-9e51-c6163c9dbd05.png)

**Comment by saleiva** [comment]
There are a few options that doesn't make sense IMO, like private datasets, private maps and sync tables.

**Comment by juanignaciosl** [comment]
@saleiva please agree with @andrewxhill the features that an organization owner should be able to set and I'll add and remove as needed from this branch (it didn't get to master).

**Comment by saleiva** [comment]
I've already opened another ticket for that. I'd keep the scope of this as small as we can

**Comment by juanignaciosl** [comment]
Ok, you say we should remove "private maps, private datasets and sync tables", so only soft limits should be added, isn't it? User quota has been there since organization signup.

**Comment by saleiva** [comment]
yep

**Comment by juanignaciosl** [comment]
Ok.

**Comment by andrewxhill** [comment]
sure, all saleiva says i agree with. but regarding the rest, there is something missing. let's take one example,

* Extra tweets

the real setting should be

* Limit, Soft limit, No limit {{3 way option}}
* Tweet quota {{some number input available if limit or soft limit}}

**Comment by saleiva** [comment]
Closing since this is being applied on the redesign.

**Comment by juanignaciosl** [comment]
Will be finished after #4878.

**Comment by juanignaciosl** [comment]
@MariaCheca ping me if you need this for the redesign #4880

**Comment by ethervoid** [comment]
We are going to need this in order to define the [geocoder API configuration](https://github.com/CartoDB/cartodb/issues/6198) due we need to store in some way the default organization value and the exceptions to that general rule.

**Comment by juanignaciosl** [comment]
Closed because soft limits for geocodings, tweets and isolines can be set by owner.
